sound/soc/sof/ipc4-pcm.c

```diff
@@ -294,6 +296,15 @@ static int sof_ipc4_chain_dma_trigger(struct snd_sof_dev *sdev,
 			msg.extension |= pipeline->msg.extension;
 	}
 
+	if (direction == SNDRV_PCM_STREAM_CAPTURE) {
+		/*
+		 * for capture we need to add num_playback to the value (stream_tag - 1)
```

Comment:
I would reword it to something like this:
```c
		/*
		 * For ChainDMA the DMA ids are unique with the following mapping:
		 * playback:  0 - (num_playback_streams - 1)
		 * capture:   num_playback_streams - (num_playback_streams +
		 *				      num_capture_streams - 1)
		 *
		 * Add the num_playback_streams offset to the DMA ids stored in
		 * msg.primary in case capture
		 */
```

Reply:
thanks for the suggestion, will add.

Reply:
done